mods/d2k/rules/structures.yaml

```diff
@@ -649,6 +649,9 @@ wall:
 	Sellable:
 		SellSounds: CHUNG.WAV
 	Guardable:
+	Explodes:
+		Weapon: WallExplode
+		EmptyWeapon: WallExplode
 	ThrowsShrapnel:
```

Comment:
I didn't notice shrapnel when I tested the original, nor does it make sense (concrete walls don't contain anything combustable).  Can you remove this?